<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor de Tareas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QTabWidget" name="Pestanas">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_tareas">
         <attribute name="title">
          <string>Tareas</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxEtiqueta"/>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QTextEdit" name="TextDescripcion">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QTableWidget" name="TablaTareas">
              <column>
               <property name="text">
                <string>Nombre</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fecha</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Terminada</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="comboBoxCategorias"/>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineEdit">
              <property name="text">
               <string>Categorias</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="text">
               <string>Etiqueta</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_categorias">
         <attribute name="title">
          <string>Categorias</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTableWidget" name="TablaCategorias">
              <column>
               <property name="text">
                <string>Nombre</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Descripci√≥n</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_etiquetas">
         <attribute name="title">
          <string>Etiquetas</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="1" column="0">
             <widget class="QTableWidget" name="TablaEtiquetas">
              <column>
               <property name="text">
                <string>Tipo</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuCategorias">
    <property name="title">
     <string>Categorias</string>
    </property>
    <addaction name="actionNuevaCategoria"/>
   </widget>
   <widget class="QMenu" name="menuEtiquetas">
    <property name="title">
     <string>Etiquetas</string>
    </property>
    <addaction name="actionNuevaEtiqueta"/>
   </widget>
   <widget class="QMenu" name="menuTareas">
    <property name="title">
     <string>Tareas</string>
    </property>
    <addaction name="actionNuevaTarea"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuTareas"/>
   <addaction name="menuCategorias"/>
   <addaction name="menuEtiquetas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevaTarea">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionNuevaCategoria">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionNuevaEtiqueta">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
